### PR TITLE
Add browsing data sample

### PR DESCRIPTION
**Pull request by ace-n**

### CODE REVIEW

**Comment by jmdobry** [comment]
This branch didn't get rebased properly, it overwrites a lot of the copyTable code.

File: bigquery/tables.js [hunk 1]
```diff
@@ -252,7 +266,7 @@ cli
   .command('create <dataset> <table>', 'Create a new table in the specified dataset.', {}, function (options) {
     program.createTable(utils.pick(options, ['dataset', 'table']), utils.makeHandler());
   })
-  .command('list <dataset>', 'List tables in the specified dataset.', {}, function (options) {
+  .command('tables <dataset>', 'List tables in the specified dataset.', {}, function (options) {
```
Comment (jmdobry) [inline bigquery/tables.js]: This should stay `list` to be consistent with all the other sample CLIs.

Comment (ace-n) [inline bigquery/tables.js]: So the problem is that we can `list` both tables and rows - should I use something like `list_tables` and `list_rows` instead?

Comment (jmdobry) [inline bigquery/tables.js]: I was thinking of having `list` for listing tables, since the sample is `tables.js`, and `browse` for the rows, which matches the sample description "Browse a table.".

```
node tables list my_dataset
node tables browse my_dataset my_table
```

Comment (jmdobry) [inline bigquery/tables.js]: Or

```
node tables list my_dataset
node tables rows my_dataset my_table
or
node tables getRows my_dataset my_table
or
node tables get-rows my_dataset my_table
or
node tables listRows my_dataset my_table
or
node tables list-rows my_dataset my_table
```

I don't really mind what the CLI command is for running `listRows`, but I think the command for listing tables should be `list` for consistency.

Comment (ace-n) [inline bigquery/tables.js]: Point taken, will fix.

File: bigquery/system-test/tables.test.js [hunk 2]
```diff
@@ -63,20 +63,17 @@ describe('bigquery:tables', function () {
...
+    bigquery.dataset(options.dataset).delete({ force: true }, function () {
+      // Delete files
+      storage.bucket(options.bucket).deleteFiles({ force: true }, function (err) {
+        if (err) {
+          return done(err);
+        }
+        // Delete bucket
+        setTimeout(function () {
+          storage.bucket(options.bucket).delete(done);
+        }, 2000);
```
Comment (jmdobry) [inline bigquery/system-test/tables.test.js]: Why these changes? Don't we still need to delete both `srcDataset` _and_ `destDataset`?

File: bigquery/system-test/tables.test.js [hunk 3]
```diff
@@ -191,6 +188,20 @@ describe('bigquery:tables', function () {
     });
   });
 
+  describe('browseRows', function () {
+    it('should display rows in a table', function (done) {
+      program.browseRows(options.dataset, options.table, function (err, rows) {
+        assert.equal(err, null);
+        assert.notEqual(rows, null);
```
Comment (jmdobry) [inline bigquery/system-test/tables.test.js]: If we're asserting that `rows` is an array (asserting a type), then I think it's redundant to assert that `rows` is not `null` (asserting a type).

File: bigquery/tables.js [hunk 4]
```diff
@@ -80,6 +80,19 @@ function listTables (options, callback) {
 }
 // [END list_tables]
 
+function browseRows (dataset, table, callback) {
+  var bigquery = BigQuery();
+  var tableObj = bigquery.dataset(dataset).table(table);
+  tableObj.getRows(function (err, rows) {
```
Comment (jmdobry) [inline bigquery/tables.js]: Add a space between `tableObj` and `table.getRows` for readability. Also, add the following above `tableObject.getRows`:

```
// See https://googlecloudplatform.github.io/google-cloud-node/#/docs/bigquery/latest/bigquery/table?method=getRows
```

**Comment by jmdobry** [comment]
LGTM after comments are addressed.

**Comment by ace-n** [comment]
@dpebot merge when green
